tests/flow/base.py

```diff
@@ -0,0 +1,49 @@
+import unittest
+import redis
+
+
+class FlowTestsBase(unittest.TestCase):
+    RUN_TIME_ROW_INDEX = 4
+
+    @classmethod
+    def setUpClass(cls):
+        cls.redis_con = redis.Redis(host='localhost', port=6379)
+
+    def _assert_only_expected_resuls_are_in_actual_results(self,
+                                                           actual_result,
+                                                           query_info):
+        # assert num resuls
+        num_results = len(actual_result.result_set[1:])
+        self.assertEqual(num_results, len(query_info.expected_result))
+
+        # assert actual values vs expected values
+        for res in query_info.expected_result:
+            self.assertTrue(res in actual_result.result_set[1:],
```

Comment:
declare actual_result_set = actual_result.result_set[1:]
before entering the loop